content/docs/15.how-to-guides/debezium.md

```diff
@@ -0,0 +1,526 @@
+---
+title: Debezium
```

Comment:
Based on the issue, this is specifically for the triggers. Can we rename the title to reflect this?

Reply:
While this guide is generic, we should specify what this enables you to do in Kestra